Core/include/Acts/Vertexing/AdaptiveMultiVertexFinder.hpp

```diff
@@ -150,13 +150,6 @@ class AdaptiveMultiVertexFinder {
     // Default fitQuality for constraint vertex in case no beamspot
     // constraint is provided
     std::pair<double, double> defaultConstrFitQuality{0., -3.};
-
-    // Use the full available vertex covariance information after
-    // seeding for the IP estimation. In original implementation
-    // this is not (!) done, however, this is probably not correct.
-    // So definitely consider setting this to true.
-    bool useVertexCovForIPEstimation = false;
```

Comment:
So this was previously already the default, meaning that with the change, if you were using the default implementation, there should be no changes. As far as I can tell, the default in Athena is also `false` and is not configured to `true` anywhere, so I think there should be no output changes in Athena from this.

Reply:
It seems to have an effect on Athena though since it does not compile, right?